docs/sites.yml

```diff
@@ -7530,3 +7530,14 @@
   categories:
     - Education
     - Learning
+- title: blog.kwst.site
+  main_url: https://blog.kwst.site
+  url: https://blog.kwst.site
+  description: 大名ではたらくフロントエンジニアのブログ
```

Comment:
Could you add an English translation to this description as well? We want to make sure the showcase is accessible for "all" users.

Reply:
OK. I understand. I'll translate it. Thanks.